### ISSUE TITLE
Force init in Gitea when repo is created

### ISSUE THREAD

**Issue by s-nt-s** [go-gitea/gitea#19093]
### Feature Description

When you create a repo in Gitea from the website you can mark "Initialize Repository (Adds .gitignore, License and README)", and when you create a repo in Gitea from the [API](https://servicio.mapa.gob.es/git/api/swagger#/organization/createOrgRepo) you can set auto_init to true.

Is there any way to configure Gitea to force Initialize Repository for all repos no matter if they are created from the website or API and no matter what the user chose for this option?

In other words, I need to configure my Gitea to always initialize every repo that is created.

**Comment by lunny** [comment]
But somebody may init the repo in local and push them to Gitea.
